tools/github-team-user-store/GitHubTeamUserStore/GitHubTeamUserStore/Constants/StorageConstants.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHubTeamUserStore.Constants
+{
+    internal class StorageConstants
+    {
+        public const string AzureBlobAccountName = "azuresdkartifacts";
```

Comment:
These should not be hard-coded this should be a configuration that we pass into the tool when we run it.

Reply:
I've added command line (via System.CommandLine) arguments which will still use these values as defaults. I've renamed the class to DefaultStorageConstants and prepended each const name with "Default". This will be updated with the next commit.

Reply:
The storage constants are now gone, the URI will be a command line option and BlobUriBuilder will parse the options